py/server/deephaven/numpy.py

```diff
@@ -95,7 +95,7 @@ def to_numpy(table: Table, cols: List[str] = None) -> np.ndarray:
         if table.is_refreshing:
             table = table.snapshot()
 
-        col_def_dict = {col.name: col for col in table.columns}
+        col_def_dict = table.definition
```

Comment:
Inline and dump the var?

Reply:
The `col_def_dict` is more indicative that it is explicitly being used as a dict (and it's also less code change). Looking at the method, I think there is room for further improvements, but I don't want to muck with any of this logic atm.